mock/server.py

```diff
@@ -7,22 +7,30 @@
 #
 # Copyright Contributors to the Zowe Project.
 #
-
+import os
 from flask import Flask
 from flask import request
 import base64
 from flask import make_response
-
+import math
 app = Flask(__name__)
 
+
+
 ###  TODO:
 #RBAC routes for permissions
 #Permissions changed to array [7,7,7] to better represent all the different types of permissoins and add permission check function
 
 global_username = "mock"
 global_password = "pass"
 global_password_expired = False
-
+#set port number that server uses,could be change to any number
```

Comment:
While I agree with the isHaMode change (and at a glance, it looks like it will work 👍 ) this comment isn't actually true. We can set the port number yes, but I don't think the server will respect it, because we set the variable but we don't actually tell the server to use the variable. Most likely from 
https://stackoverflow.com/questions/20212894/how-do-i-get-flask-to-run-on-port-80
you will want to change
`if __name__ == '__main__':
    app.run(debug=True)`
    (last 2 lines of server.py) to something like 
    (maybe w/ small variation, have to test)
    `if __name__ == '__main__':
    app.run(port=global_port, debug=True)`